src/vs/base/browser/ui/breadcrumbs/breadcrumbsWidget.ts

```diff
@@ -357,4 +365,18 @@ export class BreadcrumbsWidget {
 			}
 		}
 	}
+
+	private _onCrumbDrag(event: DragEvent): void {
+		if (!this._enabled) {
+			return;
+		}
+
+		if (!event.target || event.target instanceof HTMLDivElement === false) {
+			return;
+		}
+
+		const index = this._nodes.indexOf(event.target as HTMLDivElement);
+		const payload = event as any;
```

Comment:
Not sure I like this. Maybe have a special event type for DND and be more defensive, e.g return early when index is < 0 etc